### PR TITLE
Added remaining network prep and plotting functions

### PR DESCRIPTION
**Pull request by hayesrebecca**

### CODE REVIEW

File: src/transit_equity/networks/network_prep.py [hunk 1]
```diff
@@ -13,12 +13,23 @@
...
+add_stop_level_network_metrics(gdf)
+    Calculate and add stop-level network metrics (degree centrality and eigenvector centrality) 
+    to a GeoDataFrame representing a transit network.
 """
 import os
 import pandas as pd
 import geopandas as gpd
 from sqlalchemy import and_, create_engine
 from sqlalchemy.orm import sessionmaker
+import networkx as nx
```
Comment (himanshunaidu) [inline src/transit_equity/networks/network_prep.py:32]: We need to add all the dependencies to the toml file.

File: src/transit_equity/networks/network_prep.py [hunk 2]
```diff
@@ -13,12 +13,23 @@
...
 from transit_equity.geospatial.format_conversions import load_wkb
```
Comment (himanshunaidu) [inline src/transit_equity/networks/network_prep.py]: We should use relative imports inside the public repo.

File: src/transit_equity/networks/network_plotting.py [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+>>> gdf['distance_km'] = gdf.apply(calculate_geodesic_distance_km, axis=1)
+>>> gdf['node_radius'] = 
+    gdf.apply(lambda row: calculate_node_radius(gdf, 'value_col', row['value_col'], 5, 15), axis=1)
+>>> colormap = create_colormap_scaled_to_var(gdf, 'distance_km', 'Trip Distance (km)')
+>>> network_map = 
+    plot_network_edges(gdf_trips, edge_color_var='distance_km', edge_color_caption='Distance (km)')
+"""
+from geopy.distance import geodesic
+from branca.colormap import linear
```
Comment (himanshunaidu) [inline src/transit_equity/networks/network_plotting.py:48]: We should add geopy and branca to dependencies.

**Comment by hayesrebecca** [comment]
done!

**Review by himanshunaidu — APPROVED** [review]

**Pull request merged**